compile-solidity.sh

```diff
@@ -1,14 +1,16 @@
 #!/usr/bin/env bash
+
+# TODO find a way to specify node_modules as an additional source root for solc
+# TODO or migrate to truffle and the entire compilation workflow will be changed
```

Comment:
Sounds interesting. I'm not sure if district would appreciate saying fu to clojure and infiltrating some javascript tests, but it may help us internally.

Reply:
I had 1on1 with Miso Petro today. I am gonna ask a few questions in district so I can ask about this as well. 

I would not say fu to clojure tests as they are quite good, but debugging JS tests is a bit easier and faster to write. I could live with having both (some in cljs some in JS with nicer debugging support). If there is an issue you have troubles debugging you can write a quick JS test specifically for that and have easier time debugging...

Reply:
Actually, this is not that important question as how to deploy the app to testnet/mainnet...
I will ask that later, when the more important Qs gets answered first.